pkg/beaconproxy/analyzer.go

```diff
@@ -65,42 +66,32 @@ func (a *analyzer) start() {
 			// set up beacon writer output
 			output := &update{}
 
-			// create selector pair object
-			selectorPair := entry.Hosts
-
-			// create query
-			query := bson.M{}
-
-			// if beacon has turned into a strobe, we will not have any timestamps here,
-			// and need to update beaconProxy table with the strobeFQDN flag.
+			// if uconnproxy has turned into a strobe, we will not have any timestamps here,
+			// and we need to update uconnproxy table with the strobe flag. This is being done
+			// here and not in uconnproxy because uconnproxy doesn't do reads, and doesn't know
+			// the updated conn count
```

Comment:
Are strobes ever turned back into regular unique connections if their connection counts decrement below the threshold? I think there might be a bug in the original beaconing logic regarding this as well.

Would a fix be to simply add "$set": bson.M{"strobe": false} to query when `TsList != nil` here and in the beacons package?

Reply:
Yep, that should do it! I added in the extra set field for the not-strobe case. Good catch! I updated in uconn and uconnproxy as well, since we set strobe fields there too.

Also, realized the strobe field for beaconproxy is stored as "strobe" from the RITA side and is attempted to be retrieved with strobeFQDN in the middleware. This should probably be strobeProxy in both places. I can't remember if there was a reason for not just naming it strobe. Thoughts?

In the uconnproxy collection, I am just going to keep it as "strobe" unless there are objections.

Reply:
I'm good with whatever makes the least amount of work. I don't think there was a specific reason for calling it one thing over another. I'm also okay with using `strobeFQDN` in the beaconProxy collection in addition to the beaconFQDN collection. At the end of the day, both collections represent strobes going out to FQDNs. 

However, if you want to call it strobeProxy here and issue a fix-up to the middleware that works for me too.

Reply:
Cool, we can just leave it as strobeFQDN to reduce the workload